context-free.md

```diff
@@ -2,21 +2,61 @@
 
 ##  Who is this programming language for?
 
+Context Free is for artists iterested in exploring generative art.
 
 ## What is easy to do in this language? Why is it easy?
 
+It's easy to generate art that has a lot of symmetry. One of the easiest tricks
+for generating an image is to create recursive shape rules, and this tends to
+lead to shapes with a recursive structure, which results in symmetry. The
+handful of tree-like example programs confirm this.
 
 ## What is hard to do in this language? Why is it hard?
 
+It would be hard to replicate a picture of some real object in the language.
+Encoding the information in a picture (which probably does not have too much
+symmetry) would take a lot of code in this language, whose primary strength is
+encoding symmetry in only a few lines.
```

Comment:
I think this is a very insightful way of describing the limitations of the language.